maxprocs/maxprocs.go

```diff
@@ -75,30 +75,39 @@ func Set(opts ...Option) (func(), error) {
 		o.apply(cfg)
 	}
 
-	prev := currentMaxProcs()
 	undo := func() {
```

Comment:
Nit: I would choose to name this as `undoNoop` to differentiate from the real `undo`.

Reply:
Good idea